pages/overview/what-is-jsonschema.md

```diff
@@ -2,50 +2,49 @@
 section: docs
 title: What is JSON Schema?
 ---
-JSON Schema is a declarative language for annotating and validating JSON documents' structure, constraints, and data types. It provides a way to standardize and define expectations for JSON data.
+JSON Schema is a declarative language for annotating and validating JSON instances' structure, constraints, and data types. It provides a way to standardize and define expectations for JSON data.
 <br/>
 
 
 ## How does JSON Schema work?
 
 When it comes to data exchange, JSON Schema stands out as a powerful standard for defining the structure and rules of JSON data. It uses a set of [keywords](https://json-schema.org/learn/glossary#keyword) to define the properties of your data.
 
-While JSON Schema provides the language, validating a JSON [instance](https://json-schema.org/learn/glossary#instance) against a [schema](https://json-schema.org/learn/glossary#schema) requires a JSON Schema [validator](https://json-schema.org/implementations#tools). The JSON validator checks if the JSON documents conform to the schema.
+While JSON Schema provides the language, validating a JSON [instance](https://json-schema.org/learn/glossary#instance) against a [schema](https://json-schema.org/learn/glossary#schema) requires a JSON Schema [validator](https://json-schema.org/implementations#validators). The JSON validator checks if the JSON documents conform to the schema.
 
-JSON validators are open-source and commercial tools that implement the JSON Schema specification. They make it very easy to integrate JSON Schema into projects of any size.
+JSON validators consist of both open-source and commercial tools that implement the JSON Schema specification. Such tooling enables easy integration of JSON Schema into projects of any size.
 
 ![How JSON Schema works](/img/json_schema.svg)
 
 ## Benefits of JSON Schema for Developers
 
 JSON Schema empowers developers in the following ways:
 
-* **Structured Data Description**: JSON Schema allows developers to describe the structure, constraints, and data types of existing JSON formats.
+* **Structured Data Description**: JSON Schema empower developers to use JSON to describe the structure, constraints, and data types of existing JSON data.
```

Comment:
Not everyone uses JSON to define their JSON Schema. Some use YAML, others use functional libraries with currying to construct the schema. I don't think adding "use JSON to" makes sense here.

Reply:
Thanks for the clarification Ben!